README.md

```diff
@@ -19,6 +19,12 @@ ember-cli-eslint
 Installation
 ------------------------------------------------------------------------------
 
+ESLint 4 (for Node 4 and above):
+
+```
+ember install ember-cli-eslint@4
+```
+
 ESLint 3 (for Node 4 and above):
```

Comment:
I think 2 is unnecessary, 3 maybe too given that they have the same Node version compat